package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@loopback/grpc",
-  "version": "0.2.0",
+  "version": "1.0.0",
```

Comment:
Please keep the original version number; A version bump will be done in a separate commit.